src/AddTodoForm.js

```diff
@@ -16,9 +17,7 @@ function AddToDoForm({onAddTodo}){
     }
     return(
         <form onSubmit={handleAddTodo}>
-            <label htmlFor='todoTitle'>Title </label>
-            <input type="text" id='todoTitle' name='title' value={todoTitle}
-                onChange={handlerTitleChange}/>
+        <InputWithLabel todoTitle={todoTitle} handlerTitleChange={handlerTitleChange}>Title </InputWithLabel>
```

Comment:
Excellent work, Mariia! I appreciate your efforts to enhance your application. In my view, passing props from a parent component is indeed the most effective method.